src/main/java/org/polyjdbc/core/query/InsertQuery.java

```diff
@@ -100,24 +86,11 @@ String getSequenceField() {
     public InsertQuery value(String fieldName, Object value) {
         valueNames.append(fieldName).append(", ");
         values.append(":").append(fieldName).append(", ");
-        query.setArgument(fieldName, value);
+        setArgument(fieldName, value);
         return this;
     }
 
-    /**
-     * Manually set sequenced field value.
-     */
-    public InsertQuery sequenceValue(long value) {
-        query.setArgument(sequenceField, value);
-        sequenceValueSet = true;
-        return this;
-    }
-
-    /**
-     * Returns true if sequenced field value was set manually and there is no
-     * need to use sequence generator.
-     */
-    public boolean isSequenceValueSet() {
-        return sequenceValueSet;
+    void setArgument(String fieldName, Object value) {
```

Comment:
`InsertQuery` was supposed to have fluent API, adding this delegator breaks everything - why can't it be done gracefully?